pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2977,6 +2977,8 @@ type StorageClassDeviceSet struct {
 
 // CephFilesystemSubVolumeGroup represents a Ceph Filesystem SubVolumeGroup
 // +kubebuilder:printcolumn:name="Phase",type=string,JSONPath=`.status.phase`
+// +kubebuilder:printcolumn:name="Filesystem",type=string,JSONPath=`.spec.FilesystemName`,description="Name of the CephFileSystem"
+// +kubebuilder:printcolumn:name="Pinning",type=string,JSONPath=`.status.info.pinning`
```

Comment:
IMO even we dont need to print Pinning details in the status, instead this one we can add the `Quota` to status output

Reply:
Thanks for the comment @Madhu-1, but I am afraid I am a bit lost how to get it from Quota? My understanding is that Quota is the size of the SubVolumeGroup, but please correct me if I am wrong.
Example below:
```
  spec:
    dataPoolName: ""
    filesystemName: myfs
    pinning:
      distributed: 1
    quota: 10G
```

But I guess since, only one Pinning key can be active we could do some JSONPath magic by grabbing all keys from PinningSpec, like this: `.spec.pinning.*~`. Then we do not have to take it through Status, was this what you meant? Not 100% sure this works but can test

Reply:
@NymanRobin sorry for not being clear, my suggestion was to print quota (if present) in the `kubectl get output` instead of pinning details.

Reply:
I see, sorry read it a bit to quickly! Yea we can add Quota and/or emit Pinning, what is @travisn and @parth-gr opinions on this?

Reply:
Agreed the quota is more helpful to have by default. It doesn't hurt to have the pinning setting though, it's just helpful info for `kubectl get`. What if we show pinning as extended info with `priority=1`?